Samples/src/VisualDebugger/webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require("path");
```

Comment:
I'm not convinced that using `webpack` give us any benefit. We literally have 3 files (one .ts, one .html and one .css), adding an extra build step to pack this seems too much work for what we need.

Reply:
I think we need something like Webpack or Browserify to make the npm modules we import work in the browser (`@aspnet/signalr` and `chart.js`). I tried compiling directly with `tsc` and it didn't work.